src/components/messages/WelcomeMessages.tsx

```diff
@@ -0,0 +1,92 @@
+import { ChannelType, User } from '@sendbird/chat';
+import type { GroupChannel } from '@sendbird/chat/groupChannel';
+import {
+  MessageType,
+  SendingStatus,
+  UserMessage,
+} from '@sendbird/chat/message';
+import { RefObject } from 'react';
+
+import useSendbirdStateContext from '@uikit/hooks/useSendbirdStateContext';
+import Message from '@uikit/modules/GroupChannel/components/Message';
+import { ClientUserMessage } from '@uikit/types';
+
+import { WelcomeUserMessage } from '../../const';
+import { useConstantState } from '../../context/ConstantContext';
+import { parseTextMessage, Token } from '../../utils';
+import BotMessageWithBodyInput from '../BotMessageWithBodyInput';
+import DynamicRepliesPanel from '../DynamicRepliesPanel';
+import ParsedBotMessageBody from '../ParsedBotMessageBody';
+
+interface WelcomeMessagesProps {
+  channel: GroupChannel;
+  welcomeMessages: WelcomeUserMessage[];
+  botUser: User;
+  messageCount: number;
+  lastMessageRef: RefObject<HTMLDivElement>;
+  showSuggestedReplies: boolean;
+  timestamp?: number;
+}
+
+export default function WelcomeMessages(props: WelcomeMessagesProps) {
+  const { replacementTextList } = useConstantState();
+  const {
+    channel,
+    welcomeMessages,
+    botUser,
+    messageCount,
+    lastMessageRef,
+    showSuggestedReplies,
+    timestamp,
+  } = props;
+  const lastWelcomeMessageIndex = welcomeMessages.length - 1;
+
+  const store = useSendbirdStateContext();
+  const sb = store.stores.sdkStore.sdk;
+  const createdAt = timestamp ?? Date.now(); // channel.createdAt;
+  const localMessage: UserMessage = sb.message.buildMessageFromSerializedData({
+    messageId: channel.createdAt,
+    channelUrl: channel.url,
+    channelType: ChannelType.GROUP,
+    createdAt, // FIXME: ms? or seconds? sorted by this or id?
+    sender: botUser.serialize(),
+    sendingStatus: SendingStatus.SUCCEEDED,
+    messageType: MessageType.USER,
+    message: 'a welcome message',
+    reactions: [],
+    plugins: [],
+  }) as UserMessage;
+
+  return (
+    <Message message={localMessage as ClientUserMessage} hasSeparator={true}>
```

Comment:
이왕 type assertion 할때, 한 곳에서 한번만 하면 좋을것 같습니다. (현재는 `as UserMessage` -> `as ClientUserMessage`)

```suggestion
    <Message message={localMessage as ClientUserMessage} hasSeparator={true}>
```

Reply:
done!